docs/ref/contrib/postgres/search.txt

```diff
@@ -189,3 +189,59 @@ if it were an annotated ``SearchVector``::
     [<Entry: Cheese on Toast recipes>, <Entry: Pizza recipes>]
 
 .. _PostgreSQL documentation: http://www.postgresql.org/docs/current/static/textsearch-features.html#TEXTSEARCH-UPDATE-TRIGGERS
+
+
+``TrigramSimilarity``
+=====================
+
+.. class:: TrigramSimilarity(expression, expression, **extra)
```

Comment:
Is it okay to rename this `expression, string` for consistency with the actual code or would you like some other resolution?